src/main/java/school/lemon/changerequest/java/generics/Pairs/PairUtils.java

```diff
@@ -0,0 +1,52 @@
+package school.lemon.changerequest.java.generics.Pairs;
+
+/**
+ * Created by lbrdev on 21.02.2017.
+ * Project: generics.hw1
+ */
+public final class PairUtils {
+
+    public static <K, V> boolean equals(Pair<K, V> pair1, Pair<K, V> pair2) {
+        return pair1.getKey().equals(pair2.getKey()) && pair1.getValue().equals(pair2.getValue());
+    }
+
+    public static <K extends Comparable<K>, V> int compareTo(Pair<K, V> pair1, Pair<K, V> pair2) {
+        return pair1.getKey().compareTo(pair2.getKey());
+    }
+
+    public static <K, V> V[] getValues(Pair<K, V>[] pairs) {
+        V[] valuesArray = (V[]) new Object[pairs.length];
+        for (int i = 0; i < pairs.length; i++) {
+            valuesArray[i] = pairs[i].getValue();
+        }
+        return valuesArray;
+    }
+
+    public static <K, V> K[] getKeys(Pair<K, V>[] pairs) {
+        K[] keysArray = (K[]) new Object[pairs.length];
+        for (int i = 0; i < pairs.length; i++) {
+            keysArray[i] = pairs[i].getKey();
+        }
+        return keysArray;
+    }
+
+    public static <K extends Comparable<K>, V> int countGreaterThan(Pair<K, V>[] pairs, K parameter) {
+        int counter = 0;
+        for (Pair<K, V> pair : pairs) {
+            if (parameter.compareTo(pair.getKey()) == 1) {
+                counter++;
+            }
+        }
+        return counter;
+    }
+
+    public static <K, V> boolean containsUniqueObjects(Pair<K, V>[] pairs) {
+        for (int a = 0; a < pairs.length; a++ ){
```

Comment:
1. Please format your code.
2. General names for loop variables are: i, j, k... Or just some other meaningful name like `outerCounter`, `innerCounter`. But not `a`.
3. It's better to use `a < pairs.lenth - 1`, but it's ok to do it like you.